suite/tests/CMakeLists.txt

```diff
@@ -1082,7 +1082,7 @@ set(dr_test_ops -stderr_mask 0xC ${dr_os_ops})
 
 function(runtest_cmd outcmd outops key native standalone_dr dr_ops_aux separate_script)
   # assumes tools have been built: enforced at top level
-  set(dr_ops ${dr_test_ops} ${dr_ops_aux} ${TEST_OPTIONS})
+  set(dr_ops ${dr_test_ops} ${TEST_OPTIONS} ${dr_ops_aux})
```

Comment:
> We also have to change the order TEST_OPTIONS are added to dr_ops,
> otherwise -unsafe_build_ldstex is added to late and not picked up.

s/to/too/

Could you elaborate on how this is too late?  I would think we'd want TEST_OPTIONS to go last so we can override options set on individual tests: the command-line-controlled options should generally go last for that reason.